src/services

```diff
@@ -343,7 +343,7 @@ p({
       .setOrder(new foam.core.PropertyInfo[] {foam.nanos.menu.Menu.ORDER, foam.nanos.menu.Menu.LABEL})
       .build();
   """,
-  "client": "{\"of\":\"foam.nanos.menu.Menu\",\"cacheType\":\"FULL\"}"
+  "client": "{\"of\":\"foam.nanos.menu.Menu\"}"
```

Comment:
Can you try this with TTL.  The timeout/delay is 15s.

Reply:
Changed `\"cacheType\":\"FULL\"` to `\"cacheType\":\"TTL\"`. issue still present